lighthouse-core/audits/manifest-exists.js

```diff
@@ -40,6 +40,13 @@ class ManifestExists extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
+    if (!artifacts.Manifest) {
```

Comment:
> compare against null?

as above, we can compare against null if you feel strongly about it, but null is the only falsy value you could get here (and if we added the check against null specifically and let through the other falsy values, `artifacts.Manifest.value` below would throw anyways)